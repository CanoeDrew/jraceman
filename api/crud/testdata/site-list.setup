CREATE table site(
    id string, name string, street string, street2 string,
    city string, state string, zip string, country string, phone string, fax string);

INSERT into site(id, name, city, phone) values
  ("S1", "Site One", "Anytown", "800-555-1212"),
  ("S2", "Site Two", "Somewhere", "888-555-1234");
