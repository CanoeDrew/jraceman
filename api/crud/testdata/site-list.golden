[
  {
    "ID": "S1",
    "Name": "Site One",
    "Street": null,
    "Street2": null,
    "City": "Anytown",
    "State": null,
    "Zip": null,
    "Country": null,
    "Phone": "800-555-1212",
    "Fax": null
  },
  {
    "ID": "S2",
    "Name": "Site Two",
    "Street": null,
    "Street2": null,
    "City": "Somewhere",
    "State": null,
    "Zip": null,
    "Country": null,
    "Phone": "888-555-1234",
    "Fax": null
  }
]